src/core/user/dto/UserAPIContracts.ts

```diff
@@ -531,144 +531,144 @@ export type THealthProfileRequest = {
-  has_prior_immunodeficiency_diagnosis_pfnts: boolean;
+  prior_immunodeficiency_condition_diagnosis: string;
+  prior_immunodeficiency_diagnosis_primary: boolean;
+  prior_immunodeficiency_diagnosis_due_to_medication: boolean;
+  prior_immunodeficiency_diagnosis_acquired: boolean;
+  prior_immunodeficiency_diagnosis_other_connective_tissue: string;
+  prior_immunodeficiency_diagnosis_pfnts: boolean;
 
   // Lungs & Breathing
-  has_prior_breathing_and_lung_condition_diagnosis: string;
-  has_prior_breathing_and_lung_diagnosis_asthma: boolean;
-  has_prior_breathing_and_lung_diagnosis_sleep_apnea: boolean;
-  has_prior_breathing_and_lung_diagnosis_copd_ephysema_chronic_bronchitis: boolean;
-  has_prior_breathing_and_lung_diagnosis_lung_disease: boolean;
-  has_prior_breathing_and_lung_diagnosis_other: boolean;
-  has_prior_breathing_and_lung_diagnosis_none: boolean;
-  has_prior_breathing_and_lung_diagnosis_pfnts: boolean;
+  prior_breathing_and_lung_condition_diagnosis: string;
+  prior_breathing_and_lung_diagnosis_asthma: boolean;
+  prior_breathing_and_lung_diagnosis_sleep_apnea: boolean;
+  prior_breathing_and_lung_diagnosis_copd_ephysema_bronchitis: boolean;
+  prior_breathing_and_lung_diagnosis_lung_disease: boolean;
+  prior_breathing_and_lung_diagnosis_other: boolean;
+  prior_breathing_and_lung_diagnosis_none: boolean;
+  prior_breathing_and_lung_diagnosis_pfnts: boolean;
 
   // Mental Health & Mood
   recent_little_interest_or_pleasure_doing_things: string;
   recent_feeling_down_depressed_hopeless: string;
   recent_feeling_nervous_anxious_on_edge: string;
   recent_unable_to_stop_or_control_worry: string;
 
-  has_prior_psychiatric_condition_diagnosis: string;
-  has_prior_psychiatric_gad: boolean;
-  has_prior_psychiatric_panic_disorder: boolean;
-  has_prior_psychiatric_specific_phobia: boolean;
-  has_prior_psychiatric_ocd: boolean;
-  has_prior_psychiatric_ptsd: boolean;
-  has_prior_psychiatric_social_anxiety_disorder: boolean;
-  has_prior_psychiatric_Agoraphobia: boolean;
-  has_prior_psychiatric_Depression: boolean;
-  has_prior_psychiatric_add_or_adhd: boolean;
-  has_prior_psychiatric_eating_disorder: boolean;
-  has_prior_psychiatric_autism: boolean;
-  has_prior_psychiatric_personality_disorder: boolean;
-  has_prior_psychiatric_mania_hypomania_bipolar_manic_depression: boolean;
-  has_prior_psychiatric_schizophrenia: boolean;
-  has_prior_psychiatric_substance_use_disorder: boolean;
-  has_prior_psychiatric_other_psychosis_or_psychotic_illness: boolean;
-  has_prior_psychiatric_other: boolean;
-  has_prior_psychiatric_none: boolean;
-  has_prior_psychiatric_pfnts: boolean;
+  prior_psychiatric_condition_diagnosis: string;
+  prior_psychiatric_gad: boolean;
+  prior_psychiatric_panic_disorder: boolean;
+  prior_psychiatric_specific_phobia: boolean;
+  prior_psychiatric_ocd: boolean;
+  prior_psychiatric_ptsd: boolean;
+  prior_psychiatric_social_anxiety_disorder: boolean;
+  prior_psychiatric_Agoraphobia: boolean;
```

Comment:
```suggestion
  prior_psychiatric_agoraphobia: boolean;
```

Should be fully lowercase - there was a typo on the backend and has been corrected